### PR TITLE
ISSUE #2260 - Add share button in teamspace views

### LINKED ISSUE #2260: Add the share button in Teamspace views
### Description
Adding the share button on the list of views in the teamspace area

### PR DESCRIPTION
**Pull request by m1chalkubiak**
This fixes #2260

### CODE REVIEW

File: frontend/routes/viewerGui/components/views/components/viewItem/viewItem.component.tsx [hunk 1]
```diff
@@ -138,6 +139,15 @@ export class ViewItem extends React.PureComponent<IProps, any> {
 		<Small>(Default View)</Small>
 	));
 
+	public renderViewpointNameWithShare = renderWhenTrue(() => (
+		<NameRow>
+			{this.renderViewpointName(true)}
+				<IconsGroup disabled={this.state.isDeletePending}>
+					{this.props.onShare && <StyledShareIcon onClick={this.handleShareLink} />}
```
Comment (Charence) [inline frontend/routes/viewerGui/components/views/components/viewItem/viewItem.component.tsx:146]: Can the dialog box either be made wider to fit the 260px name + share button, or make `Name` slightly shorter in the dialog box to make space for the share button.

At the moment, the share button gets pushed right by long view names and there's no spacing.

Screenshot 2020-10-30 at 13 58 05

**Review by Charence — CHANGES_REQUESTED** [review]

File: frontend/routes/viewerGui/components/views/components/viewItem/viewItem.component.tsx (hunk 1 above)
Comment (m1chalkubiak) [reply frontend/routes/viewerGui/components/views/components/viewItem/viewItem.component.tsx:146]: Thanks for spotting this one @Charence , I've already updated styling for views names in this specific case. It should look much better right now.

Zrzut ekranu 2020-11-2 o 11 16 29

**Review by carmenfan — APPROVED** [review]

**Comment by carmenfan** [comment]
@keifgwinn  can remove the deployment, thanks!

**Pull request merged**